ios/FluentUI/Table View/CenteredLabelCell.swift

```diff
@@ -63,4 +72,10 @@ open class CenteredLabelCell: UITableViewCell {
     open override func setHighlighted(_ highlighted: Bool, animated: Bool) { }
 
     open override func setSelected(_ selected: Bool, animated: Bool) { }
+
+    private struct Constants {
+        static let labelFont: UIFont = Fonts.body
+        static let paddingVertical: CGFloat = 11
+        static let defaultHeight: CGFloat = 48
```

Comment:
Why this was changed from 45 to 48?
Should it be updated in the ActionsCell too?